docs/api/brownian.md

```diff
@@ -22,3 +22,38 @@ SDEs are simulated using a Brownian motion as a control. (See the neural SDE exa
         members:
             - __init__
             - evaluate
+
+---
+
+## Levy areas
+
+Brownian controls can return certain types of Levy areas. These are iterated integrals
```

Comment:
Levy should have a diacritic: Lévy.

I don't always include this when writing informally (including in your name, I know!) but I try to get it right in persisent documentation!